src/main/java/Lesson17/Order.java

```diff
@@ -1,9 +1,12 @@
 package Lesson17;
 
-import javax.persistence.*;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
 import java.io.Serializable;
 import java.math.BigDecimal;
-import java.util.Date;
+import java.time.LocalDate;
```

Comment:
You crash solution for jdbc. It is not good

Reply:
Why crash? For jdbc I have LocalDate too. But on lesson were problems with it and I find a solution for LocalDate.